heron/executor/src/python/heron_executor.py

```diff
@@ -214,6 +215,9 @@ def init_parsed_args(self, args):
     self.ckptmgr_classpath = parsed_args.ckptmgr_classpath
     self.ckptmgr_port = parsed_args.ckptmgr_port
     self.stateful_config_file = parsed_args.stateful_config_file
+    self.healthmgr_mode = parsed_args.healthmgr_mode
+    self.healthmgr_classpath = '%s:%s' % (self.scheduler_classpath, parsed_args.healthmgr_classpath)
```

Comment:
The health manager needs repacking algorithm, state manager and scheduler client. All these components are part of scheduler classpath.